aspnetcore/signalr/hubcontext.md

```diff
@@ -54,6 +54,14 @@ app.Use(async (context, next) =>
 
 > [!NOTE]
 > When client methods are called from outside of the `Hub` class, there's no caller associated with the invocation. Therefore, there's no access to the `ConnectionId`, `Caller`, and `Others` properties.
+>
+> If your application needs to map a user to the connection ID and persist that mapping, you can do one of the following:
```

Comment:
```suggestion
> Apps that need to map a user to the connection ID and persist that mapping, can do one of the following:
```

Reply:
[Use bigger ideas, fewer words](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words) : Our modern design hinges on crisp minimalism. Shorter is always better.
[Short sentences and fragments are easier to scan and read.](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human#a-focus-on-the-customer) Prune every excess word.

What about consultants who are working on an app? This repo uses a more formal style.

Reply:
I added a note to also fix those in the original published doc that text snippet was derived from, which I will have a chance to rewrite.  Thanks.